lib/rules/jsx-curly-brace-presence.js

```diff
@@ -131,10 +131,6 @@ module.exports = {
       return containsLineTerminators(text) && text.trim() === '';
     }
 
-    function isSingleExpressionTemplateLiteral(child) {
-      return child.type === 'TemplateLiteral' && child.expressions.length === 1 && child.quasis.map((quasis) => quasis.value.raw).join('') === '';
```

Comment:
we might be able to limit this rule to `string` single template literal with typechecker, but just a bit overkill at this moment